DOCS/man/input.rst

```diff
@@ -1429,8 +1429,8 @@ IPC sees. Note that the C API has separate C-level declarations with
 ``mpv_event``, which may be slightly different.
 
 Note that events are asynchronous: the player core continues running while
-events are delivered to scripts and other clients. In some cases, you can hooks
-to enforce synchronous execution.
+events are delivered to scripts and other clients. In some cases, you can use
+hooks to enforce synchronous execution.
```

Comment:
This change is still in the commit somehow. I think you should probably rebase the entire branch against master again and get rid of the conflict.

Reply:
There aren't any conflicts though (as far as I can see).

Reply:
This change doesn't even show up in the diff, so I think we're good.

Reply:
I have reverted the merge, as I don't know how amending commits that old would look in the log.

Reply:
It shows up in the diff locally for me.

Reply:
Fixed.